packages/r/re-flex/xmake.lua

```diff
@@ -0,0 +1,44 @@
+package("re-flex")
+    set_homepage("https://www.genivia.com/doc/reflex/html")
+    set_description("A high-performance C++ regex library and lexical analyzer generator with Unicode support. Extends Flex++ with Unicode support, indent/dedent anchors, lazy quantifiers, functions for lex and syntax error reporting and more. Seamlessly integrates with Bison and other parsers.")
+    set_license("BSD-3-Clause")
+
+    add_urls("https://github.com/Genivia/RE-flex/archive/refs/tags/$(version).tar.gz",
+             "https://github.com/Genivia/RE-flex.git")
+
+    add_versions("v4.3.0", "1658c1be9fa95bf948a657d75d2cef0df81b614bc6052284935774d4d8551d95")
+
+    on_install(function (package)
+        io.writefile("xmake.lua",[[
+            add_rules("mode.debug", "mode.release")
+            set_languages("cxx11")
+            add_includedirs("include")
+            set_encodings("utf-8")
+            add_vectorexts("all")
```

Comment:
cross ci failed, because gcc vectorext api passed `-march=native` flags @waruqi

Reply:
What flags should `all` be mapped for cross platform?

Reply:
https://github.com/xmake-io/xmake/pull/5114